application/features/email.feature

```diff
@@ -213,6 +206,41 @@ Feature: Email
       | to send                 | does NOT exist | does exist      | delete_mfa        | should NOT  |
       | to send                 | does exist     | does exist      | delete_mfa        | should NOT  |
       | to send                 | does NOT exist | does NOT exist  | delete_mfa        | should      |
-      | Not to send             | does NOT exist | does NOT exist  | delete_mfa        | should NOT  |
+      | NOT to send             | does NOT exist | does NOT exist  | delete_mfa        | should NOT  |
       | to send                 | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
-      | Not to send             | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
\ No newline at end of file
+      | NOT to send             | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
+
+  Scenario Outline: When to send refresh backup codes emails
```

Comment:
Do you think it would be worth adding a line to this to specify what the fakeEmailer's `$minimumBackupCodesBeforeNag` is set to?

Reply:
Done